### PR TITLE
Add Unit test for Paste source from M365 Apps 

### PR DESCRIPTION
**Pull request by BryanValverdeU**
Add unit test using clipboard data from M365 Apps.

### CODE REVIEW

File: packages/roosterjs-editor-plugins/test/paste/e2e/pasteFromWordTest.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                '<p style="margin:0in 0in 8pt;font-size:11pt;font-family:Calibri, sans-serif">Test</p><!--{"start":[0,0,0],"end":[0,0,4]}-->',
+            htmlFirstLevelChildTags: ['P', 'P'],
+            html:
+                '<p style="margin:0in 0in 8pt;font-size:11pt;font-family:Calibri, sans-serif">Test</p><p style="margin:0in 0in 8pt;font-size:11pt;font-family:Calibri, sans-serif">asdsad</p>',
+        });
+
+        editor.paste(clipboardData);
+
+        expect(editor.getContent()).toEqual(
+            Browser.isFirefox
```
Comment (JiuqingSong) [inline packages/roosterjs-editor-plugins/test/paste/e2e/pasteFromWordTest.ts:40]: Why Firefox can generate different HTML than others?

Comment (BryanValverdeU) [reply packages/roosterjs-editor-plugins/test/paste/e2e/pasteFromWordTest.ts:40]: Not sure about it. I think that is how Firefox handles the HTML. It seems they add family name style to paragraph element. That is the only difference I see.

Comment (JiuqingSong) [reply packages/roosterjs-editor-plugins/test/paste/e2e/pasteFromWordTest.ts:40]: Can we also verify the content model inside? Then we may understand why they are different

Comment (BryanValverdeU) [reply packages/roosterjs-editor-plugins/test/paste/e2e/pasteFromWordTest.ts:40]: Added the content model tests to the new package.

Comment (BryanValverdeU) [reply packages/roosterjs-editor-plugins/test/paste/e2e/pasteFromWordTest.ts]: I am just going to check whether the function that process the content was called. 
We already have other unit tests to make sure the content is sanitized correctly. These unit tests will just make sure we do not break the logic to call the function.

**Review by JiuqingSong — APPROVED** [review]

**Pull request merged**